integrationtest/performance/orm/src/test/java/org/hibernate/search/test/performance/scenario/TestReporter.java

```diff
@@ -125,15 +124,15 @@ private static void printTaskInfo(TestContext ctx, PrintWriter out) {
 		}
 	}
 
-	private static void printEnvInfo(TestContext ctx, PrintWriter out) {
+	private static void printEnvInfo(TestContext testCtx, PrintWriter out) {
 		out.println( "" );
 		out.println( "TEST CONFIGURATION" );
 		out.println( "    measure performance : " + TestConstants.arePerformanceTestsEnabled() );
 		out.println( "    threads             : " + THREADS_COUNT );
-		out.println( "    measured cycles     : " + ctx.scenario.measuredCyclesCount );
-		out.println( "    warmup cycles       : " + ctx.scenario.warmupCyclesCount );
-		out.println( "    initial book count  : " + ctx.scenario.initialBookCount );
-		out.println( "    initial author count : " + ctx.scenario.initialAuthorCount );
+		out.println( "    measured cycles     : " + testCtx.measuredCyclesCount );
+		out.println( "    warmup cycles       : " + testCtx.warmupCyclesCount );
+		out.println( "    initial book count  : " + testCtx.initialBookCount );
+		out.println( "    initial author count : " + testCtx.initialAuthorCount );
```

Comment:
Could you s/initial author count :/initial author count:/ ? Just to keep all semicolons nicely aligned ;)